static/library/highlight_component.ts

```diff
@@ -49,6 +49,11 @@ export class DatacommonsHighlightComponent extends LitElement {
     ${unsafeCSS(tilesCssString)}
   `;
 
+  // Optional: Whether to add "Per capita" to the description
+  // Used when toggling "See per capita" checkbox on subject pages.
+  @property()
+  addPerCapita?: boolean;
```

Comment:
The highlight tile, unlike other tiles, adds dates like "(2022)" on its own instead of expecting the user to add a replacement string (I think we added this during the first UN push). If we want the per capita text to be added in the same parentheses like "(Per Capita in 2022)", the tile needs to handle it at the same time it adds the date.

We could instead edit the highlight tile to behave like the other tiles, and instead add a (${date}) in detect-and-fulfill (but not sure if that potentially breaks things on the UN side)

Reply:
Got it. What do you think about just inferring this & adding "per capita" if the denominator is present?

Reply:
Thinking about this more-how about a new attribute "subheader" to override where the date goes?
This attribute could be a format string that is set to  `"{date}"` by default, but we could override it with `"Per Capita in {date}"`?

Reply:
Following discussion offline, pulled the logic out of the highlight tile to the block level to match the other tile types. 

I've added ${date} at the block level to highlight tile titles, and added a check in the highlight tile to only add (${date}) if it's missing to maintain functionality for the UN. I'll come back to do the inferring per capita when we can add auto-title-generation to all tiles.